test/integration/targets/kinesis_stream/tasks/main.yml

```diff
@@ -0,0 +1,123 @@
+---
+
+- block:
+    - name: set up aws connection info
```

Comment:
The current recommendation is to use module_defaults rather than the aws_connection_info block:

https://docs.ansible.com/ansible/latest/dev_guide/platforms/aws_guidelines.html#aws-credentials-for-integration-tests